benchmarks/IntMap.hs

```diff
@@ -90,6 +105,28 @@ upd' xs m = foldl' (\m k -> snd $ M.updateLookupWithKey (\_ a -> Just a) k m) m
 alt :: [Int] -> M.IntMap Int -> M.IntMap Int
 alt xs m = foldl' (\m k -> M.alter id k m) m xs
 
+#if MIN_VERSION_base(4,8,0)
+newtype TestIdentity a = TestIdentity { runTestIdentity :: a }
+
+instance Functor TestIdentity where
+    fmap = coerce
+
+altFTestIdentity :: [Int] -> M.IntMap Int -> M.IntMap Int
+altFTestIdentity xs m =
+    foldl' (\m k -> runTestIdentity $ M.alterF (TestIdentity . id) k m) m xs
+
+altFIdentity :: [Int] -> M.IntMap Int -> M.IntMap Int
+altFIdentity xs m = foldl' (\m k -> runIdentity $ M.alterF (pure . id) k m) m xs
```

Comment:
Waait a minute. You're only testing *one* way of using `alterF`! Please copy over the relevant `Data.Map` benchmarks and test properly! Also, `pure . id` is the same as `pure`.....